src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/AddDefendantLitigationFriendCallbackHandlerTest.java

```diff
@@ -52,36 +67,234 @@ void shouldReturnNoError_WhenAboutToStartIsInvoked() {
+                .extracting("businessProcess")
+                .extracting("camundaEvent", "status")
+                .containsOnly(ADD_DEFENDANT_LITIGATION_FRIEND.name(), "READY");
+
+        }
+
+    }
+
+    @Nested
+    class AboutToSubmitWithMultiPartyToggleOff {
+        private LocalDateTime localDateTime;
+
+        @BeforeEach
+        private void setup() {
+            when(featureToggleService.isMultipartyEnabled()).thenReturn(false);
+        }
+
+        @Test
+        void shouldUpdateBusinessProcessToReadyWithEvent_whenInvoked_WithMultiParty_1v2_SameSolicitor() {
+            CaseData caseData = CaseDataBuilder.builder()
+                .atStateAddRespondent1LitigationFriend_1v2_SameSolicitor()
+                .build();
+            CallbackParams params = callbackParamsOf(V_1, caseData, ABOUT_TO_SUBMIT);
+
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("camundaEvent", "status")
+                .containsOnly(ADD_DEFENDANT_LITIGATION_FRIEND.name(), "READY");
+
+            assertThat(response.getData()).extracting("respondent1LitigationFriend").isNotNull();
+
+        }
+
+        @Test
+        void shouldSetRespondent2_whenRespondent1OptionIsSelected_WithMultiParty_1v2_SameSolicitor() {
+            CaseData caseData = CaseDataBuilder.builder()
+                .atStateAddRespondent1LitigationFriend_1v2_SameSolicitor()
+                .selectLitigationFriend("Respondent One: Test Respondent 1")
+                .build();
+            CallbackParams params = callbackParamsOf(V_1, caseData, ABOUT_TO_SUBMIT);
+
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("camundaEvent", "status")
+                .containsOnly(ADD_DEFENDANT_LITIGATION_FRIEND.name(), "READY");
+
+            assertThat(response.getData()).extracting("respondent1LitigationFriend").isNotNull();
+            assertThat(response.getData()).extracting("respondent2LitigationFriend").isNull();
+
+        }
+
+        @Test
+        void shouldUpdateBusinessProcessToReadyWithEvent_WhenRespondent1LitigationFriendIsSet() {
+            CaseData caseData = CaseDataBuilder.builder().addRespondent1LitigationFriend()
+                .build();
+            CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
 
             assertThat(response.getData())
                 .extracting("businessProcess")
-                .extracting("status")
-                .isEqualTo("READY");
+                .extracting("camundaEvent", "status")
+                .containsOnly(ADD_DEFENDANT_LITIGATION_FRIEND.name(), "READY");
+
```

Comment:
Can we check the outcome of the fields being set for the date and created date - to be 100% certain we are directing to the AboutToStart callback (in the interim when this code is merged, but CCD code isint) As production will still be using this `respondent1LitigationFriend` field